examples/gpio_sleep.rs

```diff
@@ -27,16 +27,27 @@ fn main() -> ! {
     let     swm    = p.SWM.split();
     let mut syscon = p.SYSCON.split();
     let mut wkt    = p.WKT.enable(&mut syscon.handle);
+    #[cfg(feature = "82x")]
+    let     gpio   = p.GPIO; // GPIO is initialized by default on LPC82x.
+    #[cfg(feature = "845")]
+    let     gpio   = p.GPIO.enable(&mut syscon.handle);
 
     // We're going to need a clock for sleeping. Let's use the IRC-derived clock
     // that runs at 750 kHz.
+    #[cfg(feature = "82x")]
     let clock = syscon.irc_derived_clock;
+    #[cfg(feature = "845")]
+    let clock = syscon.fro_derived_clock;
```

Comment:
If the clocks are the same, it might be a good idea to normalize the name. Maybe choose one of them, or just choose a new name that's descriptive.